src/Jackett.Common/Definitions/hdfans.yml

```
@@ -0,0 +1,186 @@
+---
+id: hdfans
+name: HDFans
+description: "HDFans is a CHINESE Private Torrent Tracker for MOVIES / TV / GENERAL"
+language: zh-CN
+type: private
+encoding: UTF-8
+links:
+  - https://hdfans.org/
+
+caps:
+  categorymappings:
+    - {id: 401, cat: Movies, desc: "Movies/电影"}
+    - {id: 404, cat: TV/Documentary, desc: "Documentaries/纪录片"}
+    - {id: 405, cat: TV/Anime, desc: "Animations/动漫"}
+    - {id: 402, cat: TV, desc: "TV Series/剧集"}
+    - {id: 403, cat: TV, desc: "TV Shows/综艺"}
+    - {id: 404, cat: Other, desc: "Education/教育"}
+    - {id: 406, cat: Audio/Video, desc: "MusicVideo/音乐MV"}
+    - {id: 407, cat: TV/Sport, desc: "Sports/运体"}
+    - {id: 408, cat: Audio, desc: "HQ Audio/无损音乐"}
+    - {id: 419, cat: PC/0day, desc: "Software/软件"}
+    - {id: 421, cat: PC/Games, desc: "Games/游戏"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep, imdbid]
+    movie-search: [q, imdbid]
+    music-search: [q]
```

Once you've added the additional categories as highlighted by garfield69, you need to add the book-search mode:
```suggestion
    music-search: [q]
    book-search: [q]
```